src/drive/web/modules/drive/Toolbar/components/UploadButtonItem.jsx

```diff
@@ -7,12 +7,13 @@ import { compose } from 'redux'
 
 import toolbarContainer from '../toolbar'
 
-const UploadButtonItem = ({ t, displayedFolder, isDisabled }) => (
+const UploadButtonItem = ({ t, displayedFolder, isDisabled, onUploaded }) => (
```

Comment:
Does this props is used somewhere for the `UploadButtonItem` ?

Reply:
Oui un commit plus tard. J'ai mergé les deux commits pour que ce soit plus clair.